### DISCUSSION TITLE
Uncaught The user ID is required

### DISCUSSION THREAD

**Discussion by lialeon84** [emailjs-com/help#32] [Q&A] [answered]
Keeps saying this error, Uncaught The user ID is required but my user key is on the file. What could be the issue?

**Comment by xr0master** [comment, marked as answer]
Are you using SDK or REST API?

**Reply by lialeon84** [reply]
I was using an .env file to hide the keys, the reason why it wasn't working. I put all of the keys in the contact file and that made it work.
